README.md

```diff
@@ -0,0 +1,57 @@
+This tests are prepared to execute and maintain test connected with pern.pl website. Using this script user can find out if basic functionalities work properly
+To prepare environment for work the pom file must be appropriate configured as below:
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Takie samo pytanie jak wcześniej - dlaczego to znajduje się w README?
pom.xml jest częścią repo - i to wystarczy do uruchomienia. 
Warto podać za to komendę, która pozwoli na uruchomienie testów/pojedynczego testu :)

Reply:
warto poznać też podstawowe zasady formatowania [Markdown](https://www.markdownguide.org/basic-syntax/) i
dodać nagłówki, formatowanie kodu, linków itp.